example/analysis-issues

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env node
+"use strict"
+const fs = require('fs')
+
+function usage() {
+    console.log(`usage: ${process.argv[1]} *UUID*
+
+Get MythX status on a previously-submitted analyses. *UUID* is the
+value that was given back by the previously-submitted requiest.
+
+Set environment MYTHX_PASSWORD and one of the following:
+
+* MYTHX_ETH_ADDRESS or
+* MYTHX_EMAIL_ADDRESS
+
+before using.
+
+OR
+
+Set environment MYTHX_API_KEY
+`)
+    process.exit(1)
+}
+
+require('./helper')
+
+/**********************************
+  Minimal option processing
+***********************************/
+
+const argLen = process.argv.length
+if (argLen === 3 &&
+    process.argv[2].match(/^[-]{0,2}h(?:elp)?$/)) {
+    usage()
+}
+
+let timeout = 20
+if (argLen < 3 || !process.argv[2].match(/^[0-9a-f-]+$/)) {
+    console.log("Need to pass a UUID...\n")
+    usage()
+}
+
+const uuid = process.argv[2]
+
+/**********************************
+  Authentication and option setup
+***********************************/
+
+let armletOptions = {
+    apiKey: process.env.MYTHX_API_KEY,
+    email: process.env.MYTHX_EMAIL,
+    ethAddress: process.env.MYTHX_ETH_ADDRESS,
+    password: process.env.MYTHX_PASSWORD,
+    platforms: ['armlet-example']  // client chargeback
+}
+
+if (process.env.MYTHX_ETH_ADDRESS) {
```

Comment:
Is this needed? I see `armletOptions.ethAddress` set above. Same for `armlet.email`.

Reply:
Good catch - I belive this is covered in 9e151a1